executor/executor.go

```diff
@@ -157,7 +157,7 @@ func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fR
 // Executes teardown commands
 func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, shellBin, exportFile, sourceDir string, stepExitCode int) (int, error) {
 	shargs := []string{"-e", "-c"}
-	cmdStr := "export PATH=${PATH}:/opt/sd:/usr/sd/bin SD_STEP_EXIT_CODE=" + strconv.Itoa(stepExitCode) + " && " +
+	cmdStr := "export PATH=${PATH}:/opt/sd:/usr/sd/bin:/usr/sd-static/bin SD_STEP_EXIT_CODE=" + strconv.Itoa(stepExitCode) + " && " +
```

Comment:
why not reuse existing `/usr/sd/bin` ?

Reply:
like what the name implies, need to separate static and dynamic binaries